docs/features/experiment.md

```diff
@@ -194,3 +194,54 @@ necessary metrics queries, using the `{{templates.baseline.podTemplateHash}}` an
     Rollout. This is despite the fact that the PodSpec are the same. This is intentional behavior,
     in order to allow the metrics of the Experiment's pods to be delineated and queried separately
     from the metrics of the Rollout pods.
+
+
+
+## Weighted Experiment Step with Traffic Routing
+!!! important
+    Available since v1.1
+
+A Rollout using the Canary strategy along with Traffic Routing can 
+split traffic to an experiment stack in a fine-grained manner. When
+Traffic Routing is enabled, the Rollout Experiment step allows
+traffic to be shifted to experiment pods.
+
+!!! note
+    This feature is currently available only for the SMI and ALB Traffic Routers.
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: Rollout
+metadata:
+  name: guestbook
+  labels:
+    app: guestbook
+spec:
+...
+strategy:
+  canary:
+    trafficRouting:
+      alb:
+        ingress: ingress
+        ...
```

Comment:
what are these dots for...we should have samples that users can copy and run without making any changes

Reply:
Code snippets in documentation aren't meant to be copied and run. We have separate folders for runnable examples [here](https://github.com/argoproj/argo-rollouts/tree/master/examples) and [here](https://github.com/argoproj/argo-rollouts/tree/master/docs/getting-started)

Reply:
do you have runnable examples in any of those folders? if not please add them

Reply:
Added